public/script/reset_password_script.js

```diff
@@ -0,0 +1,51 @@
+// Hide by default.
+$('#div-otp-input').hide();
+$('#div-new-password-input').hide();
+
+// Submmit user email.
+$(document).on('submit', '#form-email-input', function(e) {
+  e.preventDefault();
+  showAlertMessage("An OTP is being sent to your email.");
+  $.post('/reset-password', $('#form-email-input').serialize(), function(data) {
```

Comment:
The same pattern of code although with a different selector is implemented at many places in this file.
Can we maybe optimise it a bit ?